paddle/framework/operator.h

```diff
@@ -94,6 +94,27 @@ class OperatorBase {
 
   const VariableNameMap& Inputs() const { return inputs_; }
   const VariableNameMap& Outputs() const { return outputs_; }
+
+  const std::vector<std::string> InputsNames() const {
+    std::vector<std::string> result;
+    for (auto& kv : inputs_) {
+      for (auto& name : kv.second) {
+        result.push_back(name);
+      }
+    }
+    return result;
+  }
+
+  const std::vector<std::string> OutputsNames() const {
```

Comment:
Is that same as `OutputVars` method?

Reply:
It seems the same. I will remove this method